src/email/services/mailing.service.ts

```diff
@@ -19,4 +27,30 @@ export class MailingService {
       throw new MailSendingException();
     }
   }
+
+  public async sendMailsToSubscribers(payload: IRate): Promise<void> {
+    const { currencyCode, date, rate } = payload;
+
+    const mailMessage = this.mailHtmlTemplate(currencyCode, rate, date);
```

Comment:
Probably, the parameters for this method could also be wrapped in an object for consistency)